qemu/tests/timerdevice_boot.py

```diff
@@ -169,6 +169,15 @@ def update_clksrc(session, clksrc):
     if params["os_type"] == "linux":
         verify_timedrift(session, is_hardware=True)
 
+    if params.get("is_repeat"):
+        sleep_time = params["sleep_time"]
+        nums = int(params["nums"])
+        while nums > 0:
```

Comment:
with a repeat num, `for` is ok.
And does it need to sleep before the first time to verify the time ?

Reply:
> with a repeat num, `for` is ok.
> And does it need to sleep before the first time to verify the time ?

Yes.